### PR TITLE
Feat: Added industry select

### PR DESCRIPTION
**Pull request by danielmoll**
Bit of duplication, but just want to get the feature in initially as this will help with the react-router and hashes in url.

### CODE REVIEW

File: footer.es6 [hunk 1]
```diff
@@ -48,36 +48,32 @@ export class WinFooter extends Component {
...
+      if (content) {
+        html = (
+          <span
+            className={[
+              ...generateClassNameList(klass),
+              ...extededClasses,
+            ].join(' ')}
+            dangerouslySetInnerHTML={{ // eslint-disable-line react/no-danger
+              '__html': content,
+            }}
```
Comment (fabiosantoscode) [inline footer.es6]: You're always using dangerouslySetInnerHTML, don't you want to choose whether to do that or pass it as a regular child instead? The byline and the byline-location need to pass it as a child instead, as per the original code in line 61 and 69.

**Comment by danielmoll** [comment]
@fabiosantoscode updated. Doing a number of things at once, this also has a PR dependency on this https://github.com/economist-components/component-articletemplate/pull/28 - it's all mainly around handling hashes in the URL. Just so happens there's other bits in there too. Fun times!

**Comment by fabiosantoscode** [comment]
You're allowing two extra fields to render as pure HTML. Do you know that byline and author are HTML strings, as opposed to arbitrary strings?

**Comment by danielmoll** [comment]
Bio will be potentially supplied with with `<strong />` tags and the other two are always strings.

**Comment by fabiosantoscode** [comment]
Ok cool

**Pull request merged**